examples/demo-dts/src/core.module.ts

```diff
@@ -0,0 +1,35 @@
+import { Global, Module } from '@nestjs/common'
+import { SessionEntity } from '@/models'
+import { CoreService } from '@/core.service'
+import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm'
+import { ConnectionEntity } from '@2060.io/service-agent-nestjs-client'
+import { ConfigModule, ConfigService } from '@nestjs/config'
+
+@Global()
+@Module({
+  imports: [
+    TypeOrmModule.forFeature([ConnectionEntity, SessionEntity]),
+    TypeOrmModule.forRootAsync({
+      imports: [ConfigModule],
+      useFactory: async (configService: ConfigService): Promise<TypeOrmModuleOptions> => ({
+        type: 'postgres',
+        host: configService.get<string>('appConfig.postgresHost'),
+        port: 5432,
+        username: configService.get<string>('appConfig.postgresUser'),
+        password: configService.get<string>('appConfig.postgresPassword'),
+        database: configService.get<string>('appConfig.postgresUser'),
```

Comment:
What about calling this postgresDatabaseName?

Reply:
fixed: added `POSTGRES_DB_NAME` variable and `postgresDbName` in the config of nestjs